src/data/uma-title-requirements.csv

```diff
@@ -1,11 +1,11 @@
 "text", "translation"
-"天気[雨]で4回以上出走し、3勝する","Participate a race with rainy weather 4 times or more\nWin at least 3 of them"
-"天気[曇り]で4回以上出走し、3勝する","Participate a race with cloudy weather 4 times or more\nWin at least 3 of them"
-"天気[雪]で3回以上出走し、2勝する","Participate a race with snowy weather 3 times or more\nWin at least 2 of them"
-"GⅠに1回以上出走し、\nGⅠ未勝利のまま育成ランクC以上で育成を完了する","<size=22>Participate a GI race once or more without winning any single GI race\nComplete the training with C or higher rank</size>"
+"天気[雨]で4回以上出走し、3勝する","Participate a race with rainy weather at least 4 times\nWin 3 of them"
```

Comment:
What do you think if we write it as `≥4 times` to save space? 😂

Reply:
this is great idea, should we?

Reply:
Considering length limitations, I think it's actually good.  
Even if many text don't reach the limit yet, it still frees more room/space to breathe.

Reply:
let me know if theres more that need to be changed